### PR TITLE
NonNull & Nullable annotations can be used in all platform modules

### PR DESCRIPTION
**Pull request by hendrikebbers**
fixes https://github.com/hashgraph/hedera-services/issues/5480

### CODE REVIEW

**Review by cody-littley — APPROVED** [review]

**Review by lpetrovic05 — APPROVED** [review]

File: documentation/null-handling/README.md
Comment (hendrikebbers) [reply documentation/null-handling/README.md:3]: In general I'm with you but https://github.com/hashgraph/hedera-services/pull/5483 already changes that by provide our own common Utility for that. So we should wait until that is merged and than update the documentation.
Plus: since we now have 2 repos become 1 we need to discuss how we handle documentation anyway, see my comment [here](https://github.com/hashgraph/hedera-services/pull/5565#issuecomment-1463442397).

**Review by qnswirlds — APPROVED** [review]

**Review by netopyr — APPROVED** [review]

**Review by nathanklick — APPROVED** [review]

**Pull request merged**
